plugins/common/proxy/socks5_test.go

```diff
@@ -0,0 +1,67 @@
+package proxy
+
+import (
+	"net"
+	"testing"
+	"time"
+
+	"github.com/armon/go-socks5"
+	"github.com/stretchr/testify/require"
+)
+
+func TestSocks5ProxyConfig(t *testing.T) {
```

Comment:
You need to guard this test to be excluded in short mode as there is no guarantee of a functioning network (beside localhost)...
```suggestion
func TestSocks5ProxyConfig(t *testing.T) {
        if testing.Short() {
                t.Skip("Skipping integration test in short mode")
        }

```

Reply:
sounds reasonable, commited it straight from github